### DISCUSSION TITLE
Command line arguments for remesh_smooth() function

### DISCUSSION THREAD

**Discussion by Gowrimv** [BrunoLevy/geogram#145] [General]
the remesh_smooth() function produces a mesh with less number of points than mentioned. Passing remesh:nb_pts as command line argument does not seem to have any effect either.How can i use command line or script to get desired no of points?

**Comment by BrunoLevy** [comment]
Hi, unfortunately there is no way of exactly matching user's desired number of points: sometimes it can be more, sometimes less, depending on the structure of the restricted Voronoi diagram (internally computed by the algorithm).
